### DISCUSSION TITLE
Where do i input command lines?

### DISCUSSION THREAD

**Discussion by HappyPenguinzz** [kangyu-california/PersistentWindows#313] [Q&A] [answered]
couldn't find specifics as to where.

**Comment by kangyu-california** [comment, marked as answer]
You may edit the StartPersistWindows* task in the task scheduler 
![image](https://github.com/kangyu-california/PersistentWindows/assets/59128756/71951c1f-b1ef-4861-aaad-db3109dcd1dc)

Or create a .bat file to launch PersistentWindows.exe with command arguments.

**Reply by HappyPenguinzz** [reply]
I have it in the task schedular but can't seem to get my commands to work. "-splash=0 -hotkey_window=0 -auto_restore_missing_windows=1" I had edit action and add it to arguments as I saw u tell someone previously.

**Reply by HappyPenguinzz** [reply]
nvm i just had to close and reopen it from task scheduler 😅 but thank you so much !!! i fkn love this application so much<3
